lumaviewpro.py

```diff
@@ -619,14 +619,17 @@ def move_relative_position(
 
 
 def move_home(axis: str):
+    global version
```

Comment:
Minor comment.  To access/read `version` from inside this function, it shouldn't need to be declared as `global` here.  `global` is only needed if intending to modify `version` from within this function.